### DISCUSSION TITLE
Match all values inside the list

### DISCUSSION THREAD

**Discussion by ksromero** [sqlalchemy/sqlalchemy#6421] [Usage Questions] [answered]
**Describe your question**

Im using `in_` column expression of sqlalchemy

```
model.query.filter(
    test_table.id.in_(ids)
).one()
```

For example I pass a `list of ids [1, 5]`, there's a `1 id` on the database but there's no `5 id`, however the query still return me the result of id 1, I want it to return the exception of no result found of one()

Is there somehow a close equivalent of `_in` expression where I want all to match the values inside the list where if one value is not there, it will fail.

**Versions**
 - OS: Windows 10
 - Python: 3.8.5
 - SQLAlchemy: 1.4
 - Database: mysql
 - DBAPI: pymysql

**Comment by CaselIT** [comment, marked as answer]
This is not really a sqlalchemy question but a sql one.

I don't think there is a sql operator that does this. You may be able to do that using multiple exists.
